### ISSUE TITLE
Why normalize rewards?

### ISSUE THREAD

**Issue by ThisIsIsaac** [pytorch/examples#590]
In [line 75](https://github.com/pytorch/examples/blob/master/reinforcement_learning/actor_critic.py#L75) of actor-critic.py, there is a code that normalizes the rewards. However, we don't normalize the values returned from the critic. Why do we do this?

**Comment by vumaasha** [comment]
I guess you are referring to this line. 

`returns = (returns - returns.mean()) / (returns.std() + eps)`

Here he is preparing the performance measure J(theta) or the loss function in PyTorch as normalized returns.  PG theorem says that the gradient of the performance measure is proportional to Return * derivate of log of policy. Since we are using montecorlo the variance in the returns will be high across episodes.

By normalizing the returns, we center the mean and reduce the variance to 1 which will help the convergence of gradient ascent
